src/context/AppProvider/AppContext.tsx

```diff
@@ -309,7 +312,7 @@ export const AppProvider = ({ children }: { children: React.ReactNode }) => {
     fetchMarketData() // fetch every time assetIds change
     const interval = setInterval(fetchMarketData, 1000 * 60 * 2) // refetch every two minutes
     return () => clearInterval(interval) // clear interval when portfolioAssetIds change
-  }, [dispatch, isPortfolioLoaded, portfolioAssetIds, txHistoryStatus])
+  }, [DEFAULT_HISTORY_TIMEFRAME, dispatch, isPortfolioLoaded, portfolioAssetIds, txHistoryStatus])
```

Comment:
revert this - react is only complaining as this was previously a constant and is now coming from a function hence it things it can change

Reply:
Reverted in https://github.com/shapeshift/web/pull/2209/commits/b1ad1a4fe1bbff1d36bb0d4501807fadb5d23b6b, though it was probably more maintainable how it was (with `DEFAULT_HISTORY_TIMEFRAME` included as a dependency). Someone will make it dynamic one day and forget to update the array.